### PR TITLE
feat: Figma properties alignment breaking

### PR DESCRIPTION
**Pull request by nmerget**
## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Refactoring (fix on existing components or architectural decisions)
-   [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: __snapshots__/header/patternhub/header-overview-should-match-screenshot.png
[the remark this answers is not in the document]
Comment (nmerget) [reply __snapshots__/header/patternhub/header-overview-should-match-screenshot.png:1]: screenshots weren't updated

File: README.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ Check our migration docs for breaking changes:
 1. [Alpha ➡ Beta](https://github.com/db-ui/mono/tree/main/docs/migration/alpha-beta.md) (0.0.x➡0.x.x)
 2. [v0.2 ➡ v0.3](https://github.com/db-ui/mono/tree/main/docs/migration/v0.2.x-to-v0.3.x.md)
 3. [v0.3 ➡ v0.4](https://github.com/db-ui/mono/tree/main/docs/migration/v0.3.x-to-v0.4.x.md)
+4. [v0.4 ➡ v0.5](https://github.com/db-ui/mono/tree/main/docs/migration/v0.4.x-to-v0.5.x.md)
```
Comment (mfranzke) [inline README.md:103]:
```suggestion
4. v0.4 ➡ v0.5: no migration needed, no breaking changes
```

File: docs/migration/v0.4.x-to-v0.5.x.md
Comment (mfranzke) [inline docs/migration/v0.4.x-to-v0.5.x.md:1]: let's delete this file

File: __snapshots__/accordion-item/patternhub/accordion-item-properties-should-match-screenshot.png
Comment (mfranzke) [inline __snapshots__/accordion-item/patternhub/accordion-item-properties-should-match-screenshot.png:1]: @nmerget this screenshot now includes both a "TODO" as well as a "No description" entry within the description column. This is most likely not intended, is it ?

File: packages/components/src/components/select/select.scss [hunk 2]
```diff
@@ -43,7 +43,7 @@ $has-before-padding: calc(
 		color: colors.$db-adaptive-on-bg-basic-emphasis-100-default;
 	}
 
-	&[data-variant="hidden"] {
+	&[data-hide-label="true"] {
```
Comment (mfranzke) [inline packages/components/src/components/select/select.scss:46]: @nmerget isn't it, that we even also need to mention this aspect within the https://github.com/db-ui/mono/blob/3e026a9d28075bb1ba634f6c8d53acc1540e4bca/docs/migration/v0.5.x-to-v0.6.x.md file ?

Comment (nmerget) [reply packages/components/src/components/select/select.scss:46]: It is named `showLabel="true/false"` in design. But the label is the default so I inverted the internal `data-` attribute. The property will be `showLabel` as we see it in the migration guide

Comment (mfranzke) [reply packages/components/src/components/select/select.scss:46]: but than you could still make this `data-show-label="false"`.

**Review by mfranzke — APPROVED** [review]

**Pull request merged**